mono/mini/driver.c

```diff
@@ -2354,12 +2367,21 @@ mono_main (int argc, char* argv[])
 			mname = argv [++i];
 			mono_graph_options = MONO_GRAPH_CFG;
 			action = DO_DRAW;
+#ifndef ENABLE_NETCORE
 		} else if (strcmp (argv [i], "--debug") == 0) {
 			enable_debugging = TRUE;
+#endif
 		} else if (strncmp (argv [i], "--debug=", 8) == 0) {
 			enable_debugging = TRUE;
 			if (!parse_debug_options (argv [i] + 8))
```

Comment:
I expected to see a change to `parse_debug_options` to add an `"ignore"` check...

Reply:
Should be there?  On line 219